src/bundle/Templating/Twig/EmbeddedItemEditFormExtension.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\AdminUi\Templating\Twig;
+
+use Ibexa\AdminUi\Form\Data\Content\Draft\ContentEditData;
+use Ibexa\AdminUi\Form\Factory\FormFactory;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Routing\RouterInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class EmbeddedItemEditFormExtension extends AbstractExtension
```

Comment:
Test coverage is missing

Reply:
Test coverage: https://github.com/ibexa/admin-ui/pull/875/commits/94f674445d7f603b0e3126ce616f4759fd38840a